### PR TITLE
Small improvement on default type of report_kpi_names

### PR DESCRIPTION
**Pull request by shansfolder**
To make it explicit that input argument should be a list because

```report_kpi_names='kpi'``` will lead to ```set(report_kpi_names)==set('k', 'p', 'i)```.

### CODE REVIEW

**Comment by gbordyugov** [comment]
Great, this would certainly give an explicit hint to pass a list instead of smth (but still wouldn't prevent passing a single string in place of a list)

**Comment by gbordyugov** [comment]
I think it's a great addition, making the signature of the constructor more explicit.

Would be nice to have a unit test or two to be able catching bugs here.

File: expan/core/experiment.py [hunk 1]
```diff
@@ -30,20 +30,23 @@ class Experiment(object):
     """
     Class which adds the analysis functions to experimental data.
     """
-    def __init__(self, control_variant_name, data, metadata, report_kpi_names=None, derived_kpis=[]):
+    def __init__(self, control_variant_name, data, metadata, report_kpi_names=[], derived_kpis=[]):
         experiment_column_names = set(['entity', 'variant'])
         numerical_column_names  = set(get_column_names_by_type(data, np.number))
 
+        if type(report_kpi_names) is str:
+            report_kpi_names = [report_kpi_names]
```
Comment (gbordyugov) [inline expan/core/experiment.py:38]: yes, totally makes sense

maybe a unit test case for that?

**Review by gbordyugov — CHANGES_REQUESTED** [review]

File: expan/core/experiment.py (hunk 1 above)
Comment (shansfolder) [reply expan/core/experiment.py:38]: done

**Review by gbordyugov — APPROVED** [review]
look great!

**Pull request merged**